language/metrics-output-plugin/src/__tests__/metrics-output-plugin.test.ts

```diff
@@ -0,0 +1,316 @@
+import { test, expect, describe, beforeEach, afterEach, vi } from "vitest";
+import * as fs from "fs";
+import * as path from "path";
+import { TextDocument } from "vscode-languageserver-textdocument";
+import {
+  ReferenceAssetsWebPluginManifest,
+  Types,
+} from "@player-tools/static-xlrs";
+import { PlayerLanguageService } from "@player-tools/json-language-service";
+
+import { MetricsOutput, extractFromDiagnostics } from "../metrics-output";
```

Comment:
I would just import this from root so it doesn't complain about coverage

Reply:
Ah interesting; we should probably do this across the board then (in a separate PR).